coordinator/gscoordinator/kubernetes_launcher.py

```diff
@@ -68,6 +68,13 @@
 logger = logging.getLogger("graphscope")
 
 
+class FakeKubeResponse:
+    def __init__(self, obj):
+        import json
```

Comment:
`json` is from standard library and you can freely to import it at the top-level of this file.